fvm/blueprints/contracts.go

```diff
@@ -1,50 +1,48 @@
 package blueprints
 
 import (
+	_ "embed"
 	"encoding/hex"
 	"fmt"
 
 	"github.com/onflow/cadence"
 	jsoncdc "github.com/onflow/cadence/encoding/json"
+	"github.com/onflow/cadence/runtime/common"
 
 	"github.com/onflow/flow-go/fvm/utils"
 	"github.com/onflow/flow-go/model/flow"
 )
 
-const ContractDeploymentAuthorizedAddressesPathDomain = "storage"
-const ContractDeploymentAuthorizedAddressesPathIdentifier = "authorizedAddressesToDeployContracts"
+var ContractDeploymentAuthorizedAddressesPath = cadence.Path{
+	Domain:     common.PathDomainStorage.Identifier(),
+	Identifier: "authorizedAddressesToDeployContracts",
+}
+var ContractRemovalAuthorizedAddressesPath = cadence.Path{
+	Domain:     common.PathDomainStorage.Identifier(),
+	Identifier: "authorizedAddressesToRemoveContracts",
+}
+var IsContractDeploymentRestrictedPath = cadence.Path{
+	Domain:     common.PathDomainStorage.Identifier(),
+	Identifier: "isContractDeploymentRestricted",
+}
 
-const ContractRemovalAuthorizedAddressesPathDomain = "storage"
-const ContractRemovalAuthorizedAddressesPathIdentifier = "authorizedAddressesToRemoveContracts"
+//go:embed scripts/setContractOperationAuthorizersTransactionTemplate.cdc
```

Comment:
thank you for doing this - it also has a potential benefit that when we have a good Cadence linter in place later, all these .cdc files can be easily linted